tests/SpecTests/client-side-encryption/tests/fle2-InsertFind-Unindexed.json

```diff
@@ -241,10 +241,10 @@
             }
           },
           "result": {
-            "errorContains": "Cannot query"
+            "errorContains": "encrypt"
           }
         }
       ]
     }
   ]
-}
+}
```

Comment:
The original file in [mongodb/specifications@1a911b5](https://github.com/mongodb/specifications/blob/1a911b5e8911f6f57426ec1d451dbdc19480afd8/source/client-side-encryption/tests/legacy/fle2-InsertFind-Unindexed.json) includes a trailing new line, which this PR is removing. Please update the file.

If this was inadvertently caused by your IDE trimming the trailing newline automatically, you may need to adjust that behavior.

Reply:
Done